source/guide/3.2/plugin-aws.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: bt_wiki
+title: AWS Plugin
+category: Plugins
+publish: true
+abstract: Cloudify AWS plugin description and configuration
+pageord: 600
+---
+{%summary%}
+{%endsummary%}
+
+
+# Description
+
+The AWS plugin allows users to use an AWS based cloud infrastructure for deploying services and applications.
+For more information about the library, please refer to: [http://boto.readthedocs.org/en/latest/index.html](http://boto.readthedocs.org/en/latest/index.html).
+
+
+# Plugin Requirements
+
+* Python Versions:
+  * 2.7.x
+
+
+# Types
+
+## cloudify.aws.nodes.Instance
+
+**Derived From:** [cloudify.nodes.Compute](reference-types.html)
+
+**Properties:**
+
+  * `parameters` key-value server configuration as described in [AWS EC2 Classic](http://boto.readthedocs.org/en/latest/ref/ec2.html#module-boto.ec2.instance).
+    * **Notes:**
+      * The public key which is set for the server needs to match the private key name in your AWS account. The public key may be set in a number of ways:
+        * By connecting the server node to a keypair node using the `cloudify.aws.relationships.instance_connected_to_keypair` relationship.
+        * By setting it explicitly in the `key_name` key under the `parameters` property.
+        * If the agent's keypair information is set in the provider context, the agents' keypair will serve as the default public key to be used if it was not specified otherwise.
+      * If the server is to have an agent installed on it, it should use the agents security group. If you are using a manager bootstrapped with the standard aws-manager-blueprint, there is a provider context dictionary on the manager that provides this value to the plugin. You can also use other security groups by:
+        * security_groups: list of security group names.
```

Comment:
please verify that any `property` is ticked. so.. for instance:
`security_groups`.

Thanks!